newIDE/app/src/UI/ColorField/index.js

```diff
@@ -71,10 +72,14 @@ export default class ColorField extends React.Component<Props, State> {
   render() {
     return (
       <div
-        style={{
-          ...styles.container,
-          width: this.props.fullWidth ? '100%' : undefined,
-        }}
+        style={
+          this.props.style
```

Comment:
what style is passed to this component?
I'm wondering if this could be simplified to just a set of props, to avoid making this component's style overridden that easily

Reply:
Agree, 99% of the time we want to avoid these custom styles.

Reply:
`PropertiesEditor` passes this:
```JS
  field: {
    flex: 1,
    width: 'auto',
  },
```

Reply:
Maybe I should just add `<Column expand>` around the ColorField in the PropertiesEditor?

Reply:
Much better, thanks!